proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -720,21 +720,13 @@ message ValidatorQueue {
     // validator set.
     uint64 churn_limit = 1;
 
-    // Ordered list of 48 byte public keys awaiting activation. 0th index is the
-    // next key to be processed.
-    repeated bytes activation_public_keys = 2 [(ethereum.eth.ext.ssz_size) = "?,48", deprecated = true];
-
-    // Ordered list of public keys awaiting exit. 0th index is the next key to
-    // be processed.
-    repeated bytes exit_public_keys = 3 [(ethereum.eth.ext.ssz_size) = "?,48", deprecated = true];
-
     // Ordered list of validator indices awaiting activation. 0th item in the list is the
     // next validator index to be processed.
-    repeated uint64 activation_validator_indices = 4 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v3/consensus-types/primitives.ValidatorIndex"];
+    repeated uint64 activation_validator_indices = 2 [(ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/prysm/v3/consensus-types/primitives.ValidatorIndex"];
```

Comment:
Do not change field numbers!!

See my suggestion in your other PR https://github.com/prysmaticlabs/prysm/pull/11248#discussion_r948328529